demo/demo_app.js

```diff
@@ -49,7 +49,7 @@ render(
       <br />
       <br />
       <WSDropdown
-        text="Multiple" type="button" width="500px" placeholder="Filter values.." filterable multiple items={[
+        text="Multiple" type="button" width="500px" placeholder="Filter values.."selectAll filterable multiple items={[
```

Comment:
Does this work? there is a space missing

Reply:
it was working fine when I run the demo